datumaro/plugins/widerface_format.py

```diff
@@ -100,8 +101,10 @@ def _load_items(self, path):
                 image=image_path, annotations=annotations)
 
             try:
-                bbox_count = int(lines[line_idx + 1]) # can be the next image
-            except ValueError:
+                bbox_count = int(lines[line_idx + 1])
+            except ValueError: # can be the next image
+                continue
+            except IndexError: # the file can only contain names of images
```

Comment:
Please clarify the comments. In the second case wouldn't it be better to throw an error?

Reply:
If there are no bounding boxes, then the annotation file contains only the names of the images. And then there will be an exception for the last item. I don't think there is a need for an error here, we just skip it.